plugins/convertPathData.js

```diff
@@ -766,11 +766,18 @@ function filters(path, params, { maybeHasStrokeAndLinecap, hasMarkerMid }) {
         }
       }
 
-      // remove useless non-first path segments
-      if (params.removeUseless && !maybeHasStrokeAndLinecap) {
-        // l 0,0 / h 0 / v 0 / q 0,0 0,0 / t 0,0 / c 0,0 0,0 0,0 / s 0,0 0,0
+      // remove useless non-start/end path segments
+      var isStart = !path[index - 1] || path[index - 1].command === 'm';
+      var isEnd = !path[index + 1] || path[index + 1].command === 'm';
```

Comment:
If I understand what I'm reading correctly, this current version doesn't remove move statements at the very end of the path, which are completely useless. It should.